doc/lightning-delpay.7.md

```diff
@@ -0,0 +1,58 @@
+lightning-delpay -- Command for removing a completed payment
+============================================================
+
+SYNOPSIS
+--------
+
+**delpay** *payment_hash* \[status\] (is not specify the status is complete)
+
+DESCRIPTION
+-----------
+
+The **delpay** RPC command removes a payment as given in **listsendpays** or **listpays** with a complete or failed
+status. However, the command doesn't permit to remove the pending payment.
+
```

Comment:
DESCRIPTION section needs to describe every parameter, and describe defaults for that parameter as well if it is optional and not specified.

Reply:
Updated but the doc needs a second review in the next hours (Caused from my Dyslexia problem :smile: )